### PR TITLE
cEP-0001: Rewrite ranks and rights section

### PR DESCRIPTION
**Pull request by sims1253**
Closes: https://github.com/coala/cEPs/issues/23

This is missing some stuff from the issue as I was offline.
Also, how do I break the table line?

### CODE REVIEW

File: cEP-0001.md
Comment (Mariatta) [inline cEP-0001.md]: What do you mean by "every community member that started the newcomer guide"?

File: cEP-0001.md
Comment (Mariatta) [inline cEP-0001.md]: can be promoted to a maintainer

**Comment by sils** [comment]
remove colon after closes

File: cEP-0001.md
Comment (sils) [inline cEP-0001.md]: I'd use singular consistently

File: cEP-0001.md
Comment (sils) [inline cEP-0001.md]: contributors, typo

File: cEP-0001.md
Comment (sils) [inline cEP-0001.md:77]: uhm if we need a c level decision for demoting members that's not gonna help our "we have inactive members" problem

File: cEP-0001.md
Comment (sils) [inline cEP-0001.md]: this is a major API break, please increase to 2.0

File: cEP-0001.md
Comment (sils) [reply cEP-0001.md:77]: also dememberizations should be able to happen when breaching the CoC

Comment (sims1253) [reply cEP-0001.md:77]: But breach of CoC has do be declared by some kind of commitee (like a c level decision). That is not something objective. 
Make a C level decision twice a year to remove old maintainers, tada.

File: cEP-0001.md
Comment (sims1253) [reply cEP-0001.md]: is it? I felt like I only added new stuff but did not change old stuff.

File: cEP-0001.md
Comment (sims1253) [reply cEP-0001.md]: Uh... today I learned :D I always used `promote <rank>` awesome :)

File: cEP-0001.md
Comment (sims1253) [reply cEP-0001.md]: Hm... maybe just remove the community member. Everyone that got an invitation to the org is a newcomer.

**Comment by sils** [comment]
this shouldn't be merged before the newcomer guide is updated!

**Comment by sims1253** [comment]
blocked by https://github.com/coala/coala/pull/3657

File: cEP-0001.md
Comment (jayvdb) [inline cEP-0001.md]: capital P in Python?

File: cEP-0001.md
Comment (supergr35) [inline cEP-0001.md]: `promoted developer` -> `promoted to a developer`

File: cEP-0001.md
Comment (supergr35) [reply cEP-0001.md]: Needs to addressed

File: cEP-0001.md
Comment (supergr35) [reply cEP-0001.md]: Needs to be addressed

**Review by supergr35 — COMMENTED** [review]
Few things, some of which reminders of past reviews :)

File: cEP-0001.md
Comment (supergr35) [reply cEP-0001.md]: I agree with sils here, it seems more like most of this document has been majorly restructured to describe what the different roles can do, their voting rights, and an outline of the voting types, rather than just an outline of the voting types.

Thoughts on this?

Comment (sims1253) [reply cEP-0001.md]: Again, I don't feel like any of the old "behaviour" is actually changed but we can just make it 2.0 for the sake of making it obvious how big the change is.

Comment (supergr35) [reply cEP-0001.md]: should be ready imo after that 👍

File: cEP-0001.md
Comment (supergr35) [inline cEP-0001.md]: colon at the end of this sentence?

**Review by supergr35 — CHANGES_REQUESTED** [review]
One last thing, along with the version change:

File: cEP-0001.md
Comment (meetmangukiya) [inline cEP-0001.md]: started following/with the newcomer guide maybe?

File: cEP-0001.md
Comment (meetmangukiya) [inline cEP-0001.md]: to maintainer*

File: cEP-0001.md
Comment (supergr35) [inline cEP-0001.md]: `indefinitely` might be a better word choice than `unlimited` here 👍 after that should finally be done

**Review by supergr35 — CHANGES_REQUESTED** [review]

**Comment by sils** [comment]
please fix that thing, do a rebase then I'm happy to ack it

**Comment by sils** [comment]
ack https://github.com/coala/cEPs/pull/62/commits/37a1e32b8a083b77fea2cf7d8b45f8bd33b40303

**Comment by sils** [comment]
@rultor merge

**Pull request merged**
